<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="Deur_1_Button">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Deur 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Deur_2_Button">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Deur 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Deur_3_Button">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>80</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Deur 3</string>
    </property>
   </widget>
   <widget class="QLabel" name="Deur_1">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>160</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Deur dicht</string>
    </property>
   </widget>
   <widget class="QLabel" name="Sensor_1">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>120</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Hall Sensor1 uit</string>
    </property>
   </widget>
   <widget class="QLabel" name="Deur_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>260</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Deur dicht</string>
    </property>
   </widget>
   <widget class="QLabel" name="Deur_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>90</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Deur dicht</string>
    </property>
   </widget>
   <widget class="QPushButton" name="schuifdeurSensorKnop">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>110</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>schuifdeurSensorKnop</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="SlotDeur">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>150</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Deur3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>80</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Deur 3</string>
    </property>
   </widget>
   <widget class="QLabel" name="Deur2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>250</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Deur 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="Deur1">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>150</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Deur 1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="SlotDeur_2">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>180</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="SlotDeur_3">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>210</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="AllowButton">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>330</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Allow</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BanButton">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>330</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Ban</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameField">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>310</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
